README.md

```diff
@@ -2,9 +2,89 @@
 
 Utility for generating Astro pages from `TypeDoc`.
 
+## Usage
+
+### Basic example
+
+Minimal config:
+
+```js
+// astro.config.mjs
+
+import { defineConfig } from 'astro/config'
+import { dirname, resolve } from 'node:path'
+import { fileURLToPath } from 'node:url'
+
+import initAstroTypedoc from 'astro-typedoc'
+
+const __dirname = dirname(fileURLToPath(import.meta.url))
+
+const astroTypedoc = await initAstroTypedoc({
+  baseUrl: '/docs/',
+  entryPoints: [resolve(__dirname, 'path/to/entry/point.ts')],
```

Comment:
`path/to/entry/` → `my-library/index.ts`

Maybe we should add `// Add path to your library here`